ze_graph_ext.h

```diff
@@ -616,6 +617,20 @@ typedef ze_result_t (ZE_APICALL *ze_pfnGraphInitialize_ext_t)(
 
 // Version 1.9 ze_graph_memory_query_t reports values in bytes on Linux and Windows (previously reported in KB on Windows)
 
+///////////////////////////////////////////////////////////////////////////////
+/// @brief Extension version 1.10
+
+///////////////////////////////////////////////////////////////////////////////
+typedef ze_result_t (ZE_APICALL *ze_pfnCompilerGetSupportedOptions_ext_t)(
+    size_t* pSize,                                  ///< [in,out] pointer to the required size of the supported options string
+    char* pSupportedOptions                         ///< [in] pointer to null terminated string to return supported options
```

Comment:
Not clear how to use. If driver owns the memory this needs to be a double pointer.